edx_name_affirmation/statuses.py

```diff
@@ -59,7 +59,8 @@ def trigger_state_change_from_proctoring(cls, proctoring_status):
             'created': cls.PENDING,
             'submitted': cls.SUBMITTED,
             'verified': cls.APPROVED,
-            'rejected': cls.DENIED
+            'rejected': cls.DENIED,
+            'error': cls.DENIED
```

Comment:
Nit: If you add a trailing comma, future diffs will be cleaner, because they won't show a changed line on the previous line (.e.g. the `rejected` key, in this case).